ami_backup.py

```diff
@@ -16,84 +16,37 @@
 import datetime
 import sys
 import pprint
+import os
 
 ec = boto3.client('ec2')
-#image = ec.Image('id')
+ec2_instance_id = os.environ['instance_id']
+
 
 def lambda_handler(event, context):
-    
-    reservations = ec.describe_instances(
-        Filters=[
-            {'Name': 'tag-key', 'Values': ['backup', 'Backup', 'Snapshot']},
-        ]
-    ).get(
-        'Reservations', []
+    try:
+        retention_days = os.environ['retention']
+    except:
+        retention_days = 13
+    create_time = datetime.datetime.now()
+    create_fmt = create_time.strftime('%Y-%m-%d')
+
+    AMIid = ec.create_image(InstanceId=ec2_instance_id,
+                            Name="Lambda - " + os.environ['instance_id'] + " from " + create_fmt,
```

Comment:
Really don't like this name. `Lamda` is the strategy, but `label_id` would be much better and more descriptive. Would that be an easy change?